turbinia/processors/mount_local.py

```diff
@@ -30,6 +30,48 @@
 RETRY_MAX = 10
 
 
+def GetDiskSize(source_path):
+  """Gets the size of disk evidence in bytes.
+
+  Tries using blockdev to query the size of block devices, and falls back on
+  filesize for image files.
+
+  Args:
+    source_path(str): the source path of the disk.
+
+  Returns:
+    int: the size of the disk in bytes.
+  """
+  size = None
+
+  if not os.path.exists(source_path):
+    log.error(
+        'Cannot check disk size for non-existing source_path {0!s}'.format(
+            source_path))
+    return None
+
+  cmd = ['blockdev', '--getsize64', source_path]
```

Comment:
Yeah, I wasn't sure where we could check for it since it's not run at a task level, and we don't have a generic dependency check. Any ideas?

Reply:
@wajihyassine Any thoughts on where we could/should put a dependency check for system dependencies?  Should we create a fake `TurbiniaSystem` "Job" and put it with the rest of the dependencies?   Or have a separate system dependency config var?

Reply:
Wouldn't work under `PartitionEnumerationJob` because the job could be disabled, which would skip the dependency check.

Reply:
Thinking about this again, I think putting it in the `PartitionEnumerationJob` might be a decent compromise as it's not very likely at this point that we'll be disabling the job because there are lots of Tasks that depend on it, so if there is an issue with it, we're likely to try to resolve it ASAP.  That way we don't have to add any new features for this check in the meantime.   We could add a new feature for this, but it's not a very high priority at this point as long as we have a workaround for checking the current dependencies.

Reply:
Ack, will do.